estuary_updater/handlers/base.py

```diff
@@ -105,9 +123,14 @@ def get_or_create_build(self, identifier, original_nvr=None, force_container_lab
             log.error('Failed to get brew build using the identifier {0}'.format(identifier))
             raise
 
+        try:
+            extra_json = json.dumps(build_info['extra'])
+        except (ValueError, TypeError):
+            extra_json = {}
```

Comment:
This is incorrect. `extra_json` needs to be a string or null. I suggest making `extra_json` `None` in this case.